src/packages/@app/pages/Collections/CollectionsPage.svelte

```diff
@@ -381,16 +396,27 @@
       return response;
     }}
   />
-{/if}
+</ModalWrapperV1>
 
-{#if isImportCurlPopup}
+<!-- {#if isImportCurlPopup} -->
+<ModalWrapperV1
+  title={"Import cURL"}
+  type={"dark"}
+  width={"35%"}
+  zIndex={1000}
+  isOpen={isImportCurlPopup}
+  handleModalState={(flag = false) => {
+    isImportCurlPopup = flag;
+  }}
+>
```

Comment:
Same we did for import curl why?

Reply:
this is to handle re-renders that invokes animation every time modal opens